### PR TITLE
egress and netapp path fix

### PR DESCRIPTION
**Pull request by bridgesr**
# PR checklist

Tick what applies before you raise. Delete anything that doesn't.

## Correctness
- [x] Does what the ticket asks for
- [x] Handles the edge cases (empty, null, zero, large inputs), not just the happy path
- [x] Errors are handled, not swallowed
- [x] New logic has tests, including the failure cases
- [x] Tests assert something real, not just run the code

## Keeping it simple
- [x] Doesn't rebuild something that already exists in the codebase
- [x] No more complex or slower than it needs to be
- [x] No leftover debug logging or commented-out code

## Easy to miss (a green pipeline won't flag these)
- [x] One logical change, not a pile of unrelated stuff

### If you touched the backend
- [x] Schema changes have a migration, and the Down() doesn't drop anything it shouldn't
- [x] New app settings are wired into the fa-config templates
- [x] Secrets use Key Vault references, not plain text
- [x] New outbound HTTP clients go through the shared resilience handler (AddResiliencePolicyHandler), rather than hand-rolling retries
- [x] New calls to an external service (DDEI, Egress, NetApp) have client tests against a WireMock stub (for the serialisation, auth and error handling) and are covered by the integration tests that run against the real dev services
- [x] Ran dotnet format, since CI builds and tests but doesn't check formatting

### If you touched the UI
- [x] Ran the e2e/ suite locally (the PR build only runs the mocked tests; the full suite runs post-merge and gates the deploy)
- [x] API changes are reflected in the MSW handlers in src/mocks and the matching zod schema, so the mocked tests aren't passing against a contract that's gone
- [x] New config or feature flags are wired into the VITE build variables, not just added in code
- [x] Data fetches handle the loading, empty and error states, not just success, and match how the rest of the app already does it
- [x] Ran prettier (npm run prettier:format), since CI lints but doesn't check formatting
- [x] Checked accessibility (keyboard nav, screen reader, sensible labels), since the Lighthouse run is manual and won't flag it for you

### CODE REVIEW

File: ui-spa/src/components/case-management/transfer-materials-v1/TransferDestinationPage.tsx [hunk 1]
```diff
@@ -118,7 +118,7 @@ const TransferDestinationPage: React.FC = () => {
     const folders = [
       {
         id: netAppPath,
-        name: `Shared Drive: ${operationName}`,
+        name: `Shared Drive: ${netAppPath}`,
```
Comment (renjithabby) [inline ui-spa/src/components/case-management/transfer-materials-v1/TransferDestinationPage.tsx]: is netAppPath always a single folder name or do we need to takeout the current foldername from the netAppPath? also do we need to remove the /

Comment (bridgesr) [reply ui-spa/src/components/case-management/transfer-materials-v1/TransferDestinationPage.tsx]: @renjithabby @HasanCPS I think this is a question for the business, I thought it's useful to actually show the full path that the case is connected to as it makes it explicit to the user. For example here we are connected to Delta/Evidence, maybe stripping the last "/" makes sense but I don't think removing the Delta/ does as that suggests its connected at the top level

Capture

File: ui-spa/src/components/case-management/index.tsx [hunk 2]
```diff
@@ -100,6 +100,7 @@ const CaseManagementPage = () => {
               caseId={caseId}
               operationName={operationNameOrDefendantName}
               egressWorkspaceId={caseMetaData.egressWorkspaceId}
+              egressWorkspaceName={caseMetaData.egressWorkspaceName ?? ""}
```
Comment (HasanCPS) [inline ui-spa/src/components/case-management/index.tsx]: The schema marks this as `z.string().nullable()` and the backend has it as `string?`, so `null` is a valid value. If it comes through as null the UI will render `"Egress: "` with nothing after the colon. The old code used to fall back to `leadDefendantName` when operationName was null, so there was always something visible. We need to decide what happens here: a fallback value, a different label, or confirmation that this field is guaranteed non-null when a workspace ID exists. As it stands a user could see a broken-looking label in production.

File: ui-spa/src/mocks/data/caseMetaData.playwright.ts [hunk 3]
```diff
@@ -2,6 +2,7 @@ import { type CaseMetaDataResponse } from "../../schemas";
 export const caseMetaDataPlaywright: CaseMetaDataResponse = {
   caseId: 12,
   egressWorkspaceId: "egress_1",
+  egressWorkspaceName: "Thunderstruck",
```
Comment (HasanCPS) [inline ui-spa/src/mocks/data/caseMetaData.playwright.ts]: Every mock sets `egressWorkspaceName: "Thunderstruck"` and `operationName: "Thunderstruck"` to the same value. If someone accidentally reads `operationName` instead of `egressWorkspaceName`, every test still passes. These tests give false confidence right now. Setting them to different values (e.g. `egressWorkspaceName: "Workspace-Alpha"`, `operationName: "Thunderstruck"`) would actually catch a wrong-field bug.

**Review by HasanCPS — APPROVED** [review]
LGTM!
